guides-and-standards/guide-code-review.md

```diff
@@ -0,0 +1,144 @@
+# Code review etiquette and best practices
```

Comment:
Just a broad note, I find [this](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/) to be a good summary as well. We should probably add this as a reference at the bottom. But I'll also comment to include some of these ourselves.

Reply:
added a link in the intro.